docs/approach.rst

```diff
@@ -140,6 +140,15 @@ of the other echoes (which it is).
 .. image:: /_static/10_optimal_combination_timeseries.png
   :align: center
 
+.. note::
```

Comment:
Can we have each sentence on a newline ? Sorry !

Reply:
Done.